From 48f48d735682bb49b71c9eb7e481aec13301a1e9 Mon Sep 17 00:00:00 2001
From: Randall Puterbaugh <rputerbaugh@gmail.com>
Date: Wed, 30 Dec 2015 14:39:11 -0800
Subject: [PATCH] removed some Permissions Changes

---
 scripts/hackrobats/pushdev.sh | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/scripts/hackrobats/pushdev.sh b/scripts/hackrobats/pushdev.sh
index 709ec4c..187b88a 100755
--- a/scripts/hackrobats/pushdev.sh
+++ b/scripts/hackrobats/pushdev.sh
@@ -35,9 +35,8 @@ drush -y @$machine.dev vset maintenance_mode 0
 drush -y @$machine.dev cc all
 # Fix File and Directory Permissions on Local
 cd /var/www/$domain
-sudo chown -R deploy:www-data html/* logs/* public/* private/* tmp/*
-sudo chmod -R ug=rw,o=r,a+X logs/* private/* public/* tmp/*
-sudo chmod -R u=rw,go=r,a+X html/*
+sudo chmod -R ug=rw,o=r,a+X public/* tmp/*
+sudo chmod -R u=rw,go=r,a+X html/* logs/* private/*
 # Git steps on Local
 cd /var/www/$domain/html
 git checkout .gitignore
-- 
1.9.1

